### PR TITLE
Add game's form

### PR DESCRIPTION
**Pull request by AleksandroBoko**
The game's form was added

### CODE REVIEW

File: GameStore/Views/Game/GameForm.cshtml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿@model GameStore.Domains.Domain.GameCreationTransferModel
```
Comment (SafeNuts) [inline GameStore/Views/Game/GameForm.cshtml]: Let's create form without Razor. Just a regular javascript

Comment (AleksandroBoko) [reply GameStore/Views/Game/GameForm.cshtml]: javascript was included.

File: GameStore/Views/Game/GameForm.cshtml [hunk 2]
```diff
@@ -0,0 +1,75 @@
+﻿<script src="~/scripts/jquery-1.10.2.min.js"></script>
+<script>
+    $(document).ready(function () {
+
+        getAllGenres();
+        getAllStudios();
+
+    })
+
+    function getAllGenres() {
```
Comment (SafeNuts) [inline GameStore/Views/Game/GameForm.cshtml]: Let's move all requests to server to a single file (e.g. some client) to make code reusable.

File: GameStore/Views/Game/GameForm.cshtml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            <li><select id="studioId" name="StudioId"></select></li>
+            <li><label for="gameDate">Game production date:</label></li>
+            <li><input id="gameDate" name="Date" type="date" /></li>
+            <li><label for="gameRate">Game's rate:</label></li>
+            <li><input id="gameRate" name="Rate" type="number" /></li>
+            <li><label for="gameDescription">Description of the game</label></li>
+            <li><textarea id="gameDescription" name="Description">...Type here</textarea></li>
+            <li><label for="gameImage">Image of the game</label></li>
+            <li><input id="gameImage" name="Image" type="file" /></li>
+            <li><input id="gameSubmit" name="GameSubmit" type="submit" value="Submit"></li>
```
Comment (SafeNuts) [inline GameStore/Views/Game/GameForm.cshtml]: Event if this is suitable to hold on a single line, please, format it in a way to have each element of new row.

File: GameStore/Controllers/StudioController.cs [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    public class StudioController : ApiController
+    {
+        public StudioController(IStudioService studioService)
+        {
+            this.studioService = studioService;
+        }
+
+        private readonly IStudioService studioService;
+
+        [Route("api/studio/studios")]
```
Comment (SafeNuts) [inline GameStore/Controllers/StudioController.cs]: Specify `api/studio` as a controller prefix of a route. Not for a single one.

File: GameStore/Controllers/GenreController.cs [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    public class GenreController : ApiController
+    {
+        public GenreController(IGenreService genreService)
+        {
+            this.genreService = genreService;
+        }
+
+        private readonly IGenreService genreService;
+
+        [Route("api/genre/genres")]
```
Comment (SafeNuts) [inline GameStore/Controllers/GenreController.cs]: Same as below.

File: GameStore.Services/Services/IProducerService.cs [hunk 6]
```diff
@@ -0,0 +1,14 @@
+﻿using GameStore.Domains.Domain;
+using System;
```
Comment (SafeNuts) [inline GameStore.Services/Services/IProducerService.cs]: Remove unnecessary usings. Please, apply this to each class in further.

File: GameStore/Views/Game/GameForm.cshtml (hunk 3 above)
Comment (AleksandroBoko) [reply GameStore/Views/Game/GameForm.cshtml]: Fixed

File: GameStore/Views/Game/GameForm.cshtml (hunk 2 above)
Comment (AleksandroBoko) [reply GameStore/Views/Game/GameForm.cshtml]: Fixed

File: GameStore/Controllers/StudioController.cs (hunk 4 above)
Comment (AleksandroBoko) [reply GameStore/Controllers/StudioController.cs]: Added prefix

File: GameStore/Controllers/GenreController.cs (hunk 5 above)
Comment (AleksandroBoko) [reply GameStore/Controllers/GenreController.cs]: Fixed

File: GameStore.Services/Services/IProducerService.cs (hunk 6 above)
Comment (AleksandroBoko) [reply GameStore.Services/Services/IProducerService.cs]: Fixed

**Pull request merged**